frontend/src/features/admin-form/settings/components/SecretKeyActivationModal.tsx

```diff
@@ -202,7 +254,15 @@ export const SecretKeyActivationModal = ({
                         message: 'The secret key provided is invalid',
                       },
                     })}
-                    placeholder="Enter or upload your Secret Key to continue"
```

Comment:
Hey @SeanChinJunKai, seems like playwright is failing. I'm guessing it's likely this due to this line change here.

The fix might be to update [this particular line](https://github.com/opengovsg/FormSG/blob/develop/__tests__/e2e/helpers/createForm.ts#L198) in our playwright helpers so that it captures the right text

Reply:
I've made the change to that line, so we can run the tests again!